components/content/ProseH2.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { useRuntimeConfig } from '#imports';
```

Comment:
nitpick: this should be auto imported right?

Reply:
yeah, I missed that, will remove.

Reply:
done